invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/ExportDropdown.js

```diff
@@ -1,3 +1,9 @@
+// This file is part of InvenioRDM
+// Copyright (C) 2023 Northwestern University.
+//
+// Invenio RDM Records is free software; you can redistribute it and/or modify it
```

Comment:
```suggestion
// Copyright (C) 2023 CERN.
// Copyright (C) 2023 Northwestern University.
//
// Invenio App RDM is free software; you can redistribute it and/or modify it
```

I know the other ones have "Invenio RDM Records" but that's probably just careless copy-paste.